multiraft/multiraft.go

```diff
@@ -383,12 +410,18 @@ func (s *state) createGroup(op *createGroupOp) {
 	// HACK: for single-node groups force an immediate election instead of waiting
 	// for the randomized timeout.
 	if len(op.initialMembers) == 1 {
-		s.multiNode.Campaign(context.Background(), op.groupID)
+		//s.multiNode.Campaign(context.Background(), op.groupID)
```

Comment:
The initialMembers field is gone now so this whole section is defunct. If we want to reinstate this hack it will need to go somewhere else (but currently it doesn't seem to be necessary).